util/minetester/build_minetest.sh

```diff
@@ -0,0 +1,24 @@
+ROOT=$(pwd)
+
+mkdir -p build/normal
+mkdir -p build/headless
+
+cd build/headless
+
+
+SDL2_DIR=${ROOT}/lib/SDL/build/lib/cmake/SDL2/
+
+echo ${SDL2_DIR}
+
+cmake ../.. -DRUN_IN_PLACE=TRUE -DBUILD_HEADLESS=1 -DSDL2_DIR=${SDL2_DIR}
+make -j$(( $(nproc) > 1 ? $(nproc) - 1 : 1 )) #use max(nproc,1) threads
```

Comment:
probably not very important, but why use nproc -1 threads? also in the commend it should be max(nproc - 1, 1), right?

Reply:
My computer either bugs out or lags out pretty hard with all nproc threads, so I leave 1 thread to do other things